services/auth/authHooks.ts

```diff
@@ -0,0 +1,21 @@
+import { UseMutationOptions, useMutation } from "@tanstack/react-query";
+import {
+  CheckExchangeTokenPayload,
+  CheckExchangeTokenResponse,
+  checkExchangeToken,
+} from "./authServices";
+import { AxiosResponse } from "axios";
+import { User } from "../initChatAnonymous/initChatAnonymousServices";
```

Comment:
please change relative import to alias import
e.g
import { BannerPage } from "component/Page/BannerPage"